Remove the -mtune and -march flags from the C++ extension build command line.
diff --git a/IsoSpecPy/setup.py b/IsoSpecPy/setup.py
index 29301e1..abd8891 100644
--- a/IsoSpecPy/setup.py
+++ b/IsoSpecPy/setup.py
@@ -31,7 +31,7 @@ here = os.path.abspath(os.path.dirname(__file__))
 
 cmodule = Extension('IsoSpecCppPy',
                     sources = ['IsoSpec++/unity-build.cpp'],
-                    extra_compile_args = '-mtune=native -march=native -O3 -std=c++17'.split() #+ ['-DDEBUG']
+                    extra_compile_args = '-O3 -std=c++17'.split() #+ ['-DDEBUG']
                     )
 
 setup_args = {
